msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -188,6 +188,7 @@
         static final String ACTIVITY = "activity";
         static final String SCOPES = "scopes";
         static final String ACCOUNT = "account";
+        static final String AUTH_SCHEME = "authentication_scheme";
```

Comment:
is this used?

Reply:
Removed in d6c1d6ca7a10b33fe77dd2ad871842b49d34ee1c